### ISSUE TITLE
html tag error when using inline content

### ISSUE THREAD

**Issue by pixelbreit** [jonathanheilmann/ext-jh_magnificpopup#48]
Hi, when i use the inline type for content elements i get an additional opening and closing html tag inside my sourcecode. 
It looks something like this: 

```
...
<div class="tx-jhmagnificpopup-pi1">
<html >

 </html>
	<a href="de/startseite/" class="mfp-inline-60" data-mfp-src="#mfp-inline-60">Details</a>
	<div id="mfp-inline-60" class="white-popup-block mfp-hide">
	...
	</div>
...
```

how can i get rid of this additional html tags?
I am using jh_magnificpopup 0.6.4 with TYPO3 CMS 7.6.13

**Comment by pixelbreit** [comment]
Ok, i got it. 
If I replace the leading and ending html tags in the /jh_magnificpopup/Resources/Private/Partials/Magnificpopup directory with span, my source code is much better.
It now looks like this: 
```
...
<div class="tx-jhmagnificpopup-pi1">
<span >

 </span>
	<a href="de/startseite/" class="mfp-inline-60" data-mfp-src="#mfp-inline-60">Details</a>
	<div id="mfp-inline-60" class="white-popup-block mfp-hide">
	...
	</div>
...
```
`
Not perfect, but ok for me.

**Comment by jonathanheilmann** [comment]
Thanks for reporting this issue.

As you said, your solution works but is not perfekt.
I'm going to reopen this issue and apply a bugfix for the next version.

**Comment by jonathanheilmann** [comment]
Could be solved by https://wiki.typo3.org/TYPO3.CMS/Releases/7.3/Feature#Feature_-_.2366269:_Fluid:_Remove_ViewHelper_xmlns-attributes_and_specified_html_tag
This is done within the new version 1.0.0 I'm going to release in a few days.

**Comment by jonathanheilmann** [comment]
Resolved in version 1.0.0 uploaded just now. Sorry for this delay :/
